### ISSUE TITLE
Location of `typing.otl` and imports

### ISSUE THREAD

**Issue by davidedellagiustina** [konodyuk/obsidian-typing#10]
Just by looking at the docs, at a first glance it seems to me that the main `typing.otl` file needs to be located at the root of the Obsidian vault. In general, I think it would be a nice addition if such file could be stored in a custom location specified by the user. For instance, I'd like to save it in a `schema` folder, where I also have other plugin-related config file (e.g. templates).

In addition, once the `typing.otl` can be "confined" to some folder, it would also be nice to have the possibility to split it into different files (for example, one for each type, which would be indeed useful when type specifications grow big). I'm actually not sure whether this possibility has already been considered, since in the docs I can see a placeholder for a "Scripting > Imports" section 😄

**Comment by konodyuk** [comment]
Yes, you can configure your root schema file via `Settings > Typing > Schema Path`. And schema "splitting" is also supported via imports.

You can easily have something like this:
```
// schema/main.otl
import { Type1 } from "absolute/path/file.otl"
import { Type2 } from "./relative/path/file.otl"
```
```
// absolute/path/file.otl
type Type1 {}
```
```
// schema/relative/path/file.otl
type Type2 {}
```
Until it is properly documented, you can refer to my slightly messy yet informative [mock vault](https://github.com/konodyuk/obsidian-typing/tree/master/__mocks__/__vault__) for import examples.

**Comment by davidedellagiustina** [comment]
Thanks a lot! I didn't yet get the chance to install and try the plugin, so all my observations were based on the docs online. I'm looking forward to testing it soon!
